WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -1,27 +1,18 @@
 <div class="navbar navbar-default navbar-static-top" role="navigation">
-  <div class="ui container">
-    <div class="navbar-brand">
+  <div class="ui-container">
```

Comment:
`ui container` is actually a class(es) from Fomatic UI, while `ui-container` is not referred to anywhere, so we don't need the `ui container` classes you may just remove it altogether.